spec/features/user_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'rails_helper'
+
+RSpec.feature 'Users', type: :feature do
+  before(:all) do
+    @user = User.create(name: 'Jann Doe', bio: 'I am Jann Doe', email: 'ran.com', photo: 'https://robohash.org/hey?set=set4')
+    @post = Post.create(user_id: @user.id, title: 'Hello', text: 'Teacher need to get trained')
+    @post2 = Post.create(user_id: @user.id, title: 'Hello', text: 'My second post')
+    @post3 = Post.create(user_id: @user.id, title: 'Hello', text: 'My third post')
+  end
+
+  after(:all) do
+    @user.destroy
+    @post.destroy
+    @post2.destroy
+    @post3.destroy
+  end
+
+  it 'displays a users profile picture' do
+    visit user_path(@user)
+    expect(page).to have_css("img[src*='https://robohash.org/hey?set=set4']")
+  end
+
+  it 'displays a user UserName' do
+    visit user_path(@user)
+    expect(page).to have_content('Jann Doe')
+  end
+end
+
+# rubocop:disable Metrics/BlockLength
+RSpec.feature 'User', type: :feature do
```

Comment:
- [ ] I observed that this linter was not disabled as at last review, kindly remove it and make use of rubocop.yml file to achieve your aim. This will make your work neat. ℹ️ **Note: Line 79 inclusive**